Mimesis fails on FreeBSD

# Bug report

<!--
Hi, thanks for submitting a bug. We appreciate that.

But, we will need some information about what's wrong to help you.
-->

## What's wrong

<!-- Describe what is not working. Please, attach a traceback. -->

```
self = <mimesis.providers.path.Path object at 0x8634f1490>, platform = 'freebsd13', args = (), kwargs = {'seed': <mimesis.types._MissingSeed object at 0x845334b10>}

    def __init__(
        self,
        platform: str = sys.platform,
        *args: t.Any,
        **kwargs: t.Any,
    ) -> None:
        """Initialize attributes.
    
        Supported platforms: 'linux', 'darwin', 'win32', 'win64'.
    
        :param platform: Required platform type.
        """
        super().__init__(*args, **kwargs)
        self.platform = platform
        self._pathlib_home = PureWindowsPath() if "win" in platform else PurePosixPath()
>       self._pathlib_home /= PLATFORMS[platform]["home"]
E       KeyError: 'freebsd13'

mimesis/providers/path.py:37: KeyError
```

## How is that should be

It works

## System information

FreeBSD 13

--- a/mimesis/datasets/int/path.py
+++ b/mimesis/datasets/int/path.py
@@ -14,4 +14,7 @@ PLATFORMS = {
     "win64": {
         "home": "C:\\Users\\",
     },
+    "freebsd": {
+        "home": "/home/",
+    },
 }

--- a/mimesis/providers/path.py
+++ b/mimesis/providers/path.py
@@ -27,11 +27,13 @@ class Path(BaseProvider):
     ) -> None:
         """Initialize attributes.
 
-        Supported platforms: 'linux', 'darwin', 'win32', 'win64'.
+        Supported platforms: 'linux', 'darwin', 'win32', 'win64', 'freebsd'.
 
         :param platform: Required platform type.
         """
         super().__init__(*args, **kwargs)
+        if platform.startswith("freebsd"):
+            platform = "freebsd"
         self.platform = platform
         self._pathlib_home = PureWindowsPath() if "win" in platform else PurePosixPath()
         self._pathlib_home /= PLATFORMS[platform]["home"]

